docs/how-to-guides/develop-a-tool/customize_an_llm_tool.md

```diff
@@ -19,8 +19,17 @@ Here we use [an existing tool package](https://github.com/microsoft/promptflow/t
 
 
     @tool
-    def my_tool(connection: CustomConnection, prompt: PromptTemplate, **kwargs) -> str:
-        # Customize your own code to use the connection and prompt here.
+    def my_tool(
+        connection: CustomConnection,
+        endpoint_name: str,
```

Comment:
`endpoint_name` is not suitable to be in example.

Reply:
use `deployment_name`

Reply:
changed it. And position api before the deployment_name